streaming/java/streaming-state/src/main/java/org/ray/streaming/state/config/ConfigKey.java

```diff
@@ -0,0 +1,34 @@
+package org.ray.streaming.state.config;
+
+import java.util.Map;
+
+/**
+ * state config keys.
+ */
+public class ConfigKey {
```

Comment:
Strange name and one might `new ConfigKey()` for this utility class.

With a third-party config library what we have to add here is just options, roughly could be the Strings below.